docs/samples/topic/dynamic-sync-table.md

```diff
@@ -12,89 +12,67 @@ There are two subtle variants of dynamic sync tables. A sync table can be dynami
-      // The display name for the table, shown in the UI.
-      getName: async context => {
-        let response = await context.fetcher.fetch({
-          method: "GET",
-          url: context.sync.dynamicUrl
-        });
-        // Return whatever represents the name of the entity you're syncing.
-        return response.body.name;
-      },
-      // A label for the kind of entities that you are syncing.
-      // Using Google Sheets as example you might use "Row" here, as each synced
-      // entity is a row from the source sheet.
-      entityName: "<RowName>",
-      getSchema: async () => {
+      name: "<User-visible name for the sync table>",
+      getName: coda.makeMetadataFormula(async context => {
+        let datasourceUrl = context.sync!.dynamicUrl!;
+        // TODO: Fetch metdata about the datasource and return the name.
+        return "<Datasource Name>";
+      }),
+      getSchema: coda.makeMetadataFormula(async context => {
+        let datasourceUrl = context.sync!.dynamicUrl!;
+        // TODO: Fetch metdata about the datasource and get the list of fields.
+        let propreties = {
+          // TODO: Create a property for each field.
+        };
+        let id = "<Determine the field containing a unique ID>";
+        let primary = "<Determine the field containing the display value>";
+        let featured = [
+          // TODO: Determine which fields to show in the table by default.
+        ];
         return coda.makeSchema({
           type: coda.ValueType.Array,
           items: coda.makeObjectSchema({
             type: coda.ValueType.Object,
-            // The actual schema properties.
-            properties: {
-              // Add your properties here e.g.
-              // idColumn: {type: coda.ValueType.Number},
-              // displayColumn: {type: coda.ValueType.String},
-              // otherColumn: {type: coda.ValueType.String},
+            identity: {
+              name: "<User-visible name for the column containing the schema>",
+              dynamicUrl: datasourceUrl,
             },
-            // The property name from the properties object below that represents
-            // the unique identifier of this item. A sync table MUST have a stable
-            // unique identifier. Without one, each subsequent sync will wipe away
-            // all rows and recreate them from scratch.
-            id: "<idColumn>",
-            // The property name from the properties object below that should label
-            // this item in the UI. All properties can be seen when hovering over a
-            // synced item in the UI, but the primary property value is shown on the
-            // chip representing the full object.
-            primary: "<displayColumn>",
-            // The columns that will be displayed on the table
-            featured: ["<otherColumn>"],
+            properties: propreties,
+            id: id,
+            primary: primary,
+            featured: featured,
```

Comment:
guess this is not completed yet but usually it's like `{properties, id, primary, featured}`

Reply:
This is intentional, as per the go/sample-code-style-guide. While the shorthand notation is nice, I saw a lot of cases in the Packathon where it led to confusion.